### ISSUE TITLE
Inquiries regarding your paper on anomaly detection

### ISSUE THREAD

**Issue by CaptainBakhro** [yujiangpu20/PEL4VAD#7]
Hello Yujiang!

I have a few questions regarding how the weights obtained in your detection model can be extracted for use on a dataset of my own. Could you kindly provide guidelines on how best to do this? Your help would be much appreciated! 

Looking forward to your response. 

Best regards.

**Comment by yujiangpu20** [comment]
If you want to untilize my model on your own dataset for testing, you may follow the below steps:
1. extract I3D RGB features (shaped as T*1024)
2. generate the test.list file and rewrite the dataloader
3. modifiy the config.py and run infer.py

**Comment by CaptainBakhro** [comment]
Hello again :D @yujiangpu20 

Thank you so much for your assistance, we have managed to work our way through step 1 and part of step 2. But I had some questions regarding the dataloader. We looked through the following code lines in the main.py file (line 84-104) 

def main(cfg):
    logger = get_logger(cfg.logs_dir)
    setup_seed(cfg.seed)
    logger.info('Config:{}'.format(cfg.__dict__))
    if cfg.dataset == 'ucf-crime':
        train_data = UCFDataset(cfg, test_mode=False)
        test_data = UCFDataset(cfg, test_mode=True)
    elif cfg.dataset == 'xd-violence':
        train_data = XDataset(cfg, test_mode=False)
        test_data = XDataset(cfg, test_mode=True)
    elif cfg.dataset == 'shanghaiTech':
        train_data = SHDataset(cfg, test_mode=False)
        test_data = SHDataset(cfg, test_mode=True)
    else:
        raise RuntimeError("Do not support this dataset!")
    train_loader = DataLoader(train_data, batch_size=cfg.train_bs, shuffle=True,
                              num_workers=cfg.workers, pin_memory=True)
    test_loader = DataLoader(test_data, batch_size=cfg.test_bs, shuffle=False,
                             num_workers=cfg.workers, pin_memory=True)

Is it this segment we would have to adjust? If not, could you kindly specify which part/parts? Thanks so much once again, looking forward to your response :)
